### PR TITLE
Add tulips: star physics visualization package

### PR DESCRIPTION
**Pull request by evalaplace**

### CODE REVIEW

File: packages/tulips.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+repo: https://bitbucket.org/elaplace/tulips/src/master/
+section: domain specific libraries
+description: Visualization of the physics of stars
+site: https://astro-tulips.readthedocs.io/en/latest/# optional, default repo site
```
Comment (jklymak) [inline packages/tulips.yml:4]:
```suggestion
site: https://astro-tulips.readthedocs.io/en/latest/  # optional, default repo site
```

Comment (evalaplace) [reply packages/tulips.yml:4]: thanks for the fix

File: packages/tulips.yml [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+keywords: astronomy, visualization  # optional,
+pypi_name: astro-tulips # optional, default repo name
```
Comment (QuLogic) [inline packages/tulips.yml:6]: Don't need to keep the comments at all.
```suggestion
site: https://astro-tulips.readthedocs.io/en/latest/
keywords: astronomy, visualization
pypi_name: astro-tulips
```

**Comment by jklymak** [comment]
@evalaplace sorry, apparently we are heavily GitHub biased, and the script isn't smart enough to deal with arbitrary repo addresses.  I'll open an issue.  Sorry for the bother!

**Comment by QuLogic** [comment]
I think that problem was fixed by #156; can you update this PR?

File: packages/tulips.yml [hunk 3]
```diff
@@ -0,0 +1,5 @@
+section: domain specific libraries
```
Comment (timhoffm) [inline packages/tulips.yml:1]:
```suggestion
name: TULIPS
repo: https://bitbucket.org/elaplace/tulips/src/master/
section: domain specific libraries
```
This should be the correct content now.

**Comment by timhoffm** [comment]
I think the repo entry is correct, but our parser is wrong in complaining "package repo is not in correct format".

**Comment by QuLogic** [comment]
This just needed a rebase; CircleCI doesn't build the merge head.

**Pull request merged**